### DISCUSSION TITLE
AR model initial render size

### DISCUSSION THREAD

**Discussion by jonathan-brown** [google/model-viewer#3891] [General]
Hi there

I was wondering if someone can help me here, this is my first time using blender and .glb files. I am having trouble with the size of my model on the initial opening of the AR module. The model is blown up massively and I always need to shrink it down to 2% by pinching. Does this need changed via the file when exporting or can it be altered via <model-viewer> settings? 

I have tried to look online and can't really see where I'm going wrong. 

Thank You 

Jonathan

**Comment by elalish** [comment]
This is a very common problem because most 3D modeling software doesn't consider units. glTF are speced to always be in meters, so you probably modeled in cm and it's now 100x too big. A top-level transform to fix the units should work. If you don't want to change the GLB, you can also fix this with our scale attribute: https://modelviewer.dev/docs/index.html#entrydocs-scenegraph-attributes-scale

**Reply by jonathan-brown** [reply]
Ah brilliant! how did I miss that in the docs. Appreciate that! 

Yeah, i think I had a blender set to meters. No wonder they were the size of a building. 

I'm basically trying to build an AR poster website to showcase the art in your space. Am I right to think I can export my model to the actual size of A1, A2 and A5 in mm. I then could have a model for each poster size which will then be rendered in AR at the actual size of the poster. I then won't then need to use the scale attribute? 

Thanks.

**Reply by jonathan-brown** [reply]
Or i'm thinking i could have one default Model that is exported to say A1 then calculate the difference and use the scale attribute for the other sizes based on the original A1 size.

**Reply by elalish** [reply]
Indeed, either approach should work. 👍
